workspaces/liferay-testray-workspace/client-extensions/liferay-testray-custom-element/src/hooks/useRuns.tsx

```diff
@@ -23,19 +21,12 @@ const useRuns = () => {
 			dispatch({payload: runB, type: TestrayTypes.SET_RUN_B}),
 		[dispatch]
 	);
-	const serializedFilter = useMemo(() => {
-		return JSON.parse(searchParams.get('filter') as string) || '';
-	}, [searchParams]);
-
-	const runId = serializedFilter
-		? serializedFilter['runToCaseResult/id']
-		: null;
 
 	return {
 		compareRuns: {
 			...compareRuns,
-			runId: Number(runId),
 		},
+
```

Comment:
```suggestion
```
This space is really necessary?

Reply:
Resolved in [b0af134](https://github.com/is-solutions-delivery/liferay-portal/pull/5563/commits/b0af1340c821f9151258a2dafd5787d1321f261f), thanks for the suggestion :fire: